ts/components/screens/EdgeBorderComponent.tsx

```diff
@@ -0,0 +1,20 @@
+import { View } from "native-base";
+import * as React from "react";
+import { Image, StyleSheet } from "react-native";
+
+const styles = StyleSheet.create({
+  emptyListWrapper: {
+    alignItems: "center",
+    marginTop: 74
```

Comment:
What if we add some margin at bottom too?

### without bottom margin
<img src="https://user-images.githubusercontent.com/822471/64953582-71055100-d883-11e9-8b8e-d697df72cead.png" height="600"/>

### within bottom margin
<img src="https://user-images.githubusercontent.com/822471/64953567-6c409d00-d883-11e9-9ce9-22ae224dba5e.png" height="600"/>

Reply:
Add border in 1c107a6:

<img width="339" alt="AddMargin" src="https://user-images.githubusercontent.com/17931148/65041876-0245f780-d958-11e9-9eec-e9aaeb8ce20c.png">

Reply:
@matteodesanti : this is the result:

|iOS                          |Android                      |
|-----------------------------|-----------------------------|
|<img width="277" alt="iOS" src="https://user-images.githubusercontent.com/17931148/65048702-d5e4a800-d964-11e9-80b0-74a9fe8be087.png">|<img width="277" alt="Android" src="https://user-images.githubusercontent.com/17931148/65048315-332c2980-d964-11e9-8c48-1d266c7a5d17.png">|